src/storage/gcloud-storage/gcp-storage.service.ts

```diff
@@ -119,6 +121,41 @@ export class GCPStorageService {
         return ok(true);
     }
 
+    private async compressFile(file: File): Promise<Buffer> {
+        const chunks: Buffer[] = [];
+
+        for await (const chunk of file.stream()) {
+            chunks.push(Buffer.from(chunk));
```

Comment:
Isn't `chunk` already a buffer here?

Reply:
Yeah good one! Updated in 48acfa49443b3faa2df3813d96befa9b20390fb4